bids/utils.py

```diff
@@ -2,6 +2,11 @@
 
 import re
 import os
+from frozendict import frozendict
+
+
+# Monkeypatch to print out frozendicts *as if* they were dictionaries.
+frozendict.__repr__ = lambda s: repr({k: v for k, v in s.items()})
```

Comment:
Not particularly proud of this monkeypatch, but seemed an easy way of preventing the verbose repr of frozendicts. Happy to remove if not convincing.

Reply:
Yeah, I feel the same way—not great, but probably better than trying to come up with a more principled fix.

Reply:
Had to roll this back anyways and use a derived class.